e2e_tests/infer.sh

```diff
@@ -1750,6 +1750,16 @@ infer_json_null_graphql() {
     expect=`printf "error: failed to serialize schema\ncannot infer type of null value (see key 'is2004')"`
 }
 
+infer_json_as_null_string_graphql() {
+    output=`printf '{"myString":null}' | schema infer --graphql --as-null=string --omit-required=false 2>&1`
```

Comment:
Test looks good except we don't need `--omit-required=false` since it has no bearing on graphql schemas. When that flag is true, the "required" field is omitted from JSON schemas.